### PR TITLE
Added localized strings to editor.properties for InlineBorderRadiusEditor extension

### PR DESCRIPTION
**Pull request by feihaozi77**
@humphd I Added the strings to the edtitor.properties file for InlineBorderRadiusEditor. Please review

### CODE REVIEW

**Comment by humphd** [comment]
Assigning to @gideonthomas for review

File: locales/en-US/editor.properties [hunk 1]
```diff
@@ -208,6 +208,24 @@ PDF_FILE_TITLE=PDF File
 ################
 ## EXTENSIONS ##
 ################
+# extensions/default/InlineBorderRadiusEditor
+ALL_CORNERS=All Corners
```
Comment (gideonthomas) [inline locales/en-US/editor.properties]: Add a comment above this explaining that "All Corners" is an option that can be selected for border-radius.

File: locales/en-US/editor.properties [hunk 2]
```diff
@@ -208,6 +208,24 @@ PDF_FILE_TITLE=PDF File
...
+INDIVIDUAL_CORNERS=Individual Corners
+TL_RADIO_PX=pixels
```
Comment (gideonthomas) [inline locales/en-US/editor.properties]: All these strings are the same. Just use `PIXEL_UNIT=pixels`, `EM_UNIT=em font unit`, `PERCENTAGE_UNIT=percentage`

File: locales/en-US/editor.properties [hunk 3]
```diff
@@ -208,6 +208,24 @@ PDF_FILE_TITLE=PDF File
 ################
 ## EXTENSIONS ##
 ################
+# extensions/default/InlineBorderRadiusEditor
+ALL_CORNERS=All Corners
+INDIVIDUAL_CORNERS=Individual Corners
```
Comment (gideonthomas) [inline locales/en-US/editor.properties]: same as above

**Comment by feihaozi77** [comment]
@gideonthomas I updated the strings, I think we are almost there. Let me know if there is anything else I need to add/modify

**Pull request merged**